auth0_flutter/example/android/app/build.gradle

```diff
@@ -28,7 +28,6 @@ apply plugin: 'jacoco'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    namespace exampleAppApplicationId
```

Comment:
This was duplicated; see L32 below `if (project.android.hasProperty("namespace"))`.